config/install/printable.settings.yml

```diff
@@ -6,6 +6,7 @@ open_target_blank: TRUE
 css_include: ''
 extract_links: ''
 save_pdf: FALSE
+pdf_tool: 'wkhtmltopdf'
```

Comment:
I doubt if this is a good default because (I think) wkhtmltopdf needs a locally installed binary. Perhaps use no pdf tool as default.